### PR TITLE
Modify some text and punctuation marks

### PR DESCRIPTION
**Pull request by Atom-pro**
Submit document bug

### CODE REVIEW

**Comment by Atom-pro** [comment]
Modify some text and punctuation marks

**Comment by yorkie** [comment]
Please fix the merge conflicts at first :)

**Comment by Atom-pro** [comment]
conflicts files has resolved!

File: development/cplusplus/lumenflinger.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ m_fps：服务端的刷新频率，目前最大30fps,过快刷新灯光数据可
 
   开启：唤醒灯光服务，线程等待写入数据
   
-  关闭：此client端无法对灯光服务操作写入。当所有灯光client都关闭时，灯光服务立即进入睡眠
+  关闭：此client端无法对灯光服务操作写入。当client端所有灯光都关闭时，灯光服务立即进入睡眠
```
Comment (yorkie) [inline development/cplusplus/lumenflinger.md]: Missing spaces between CN & EN chars.

Comment (yorkie) [reply development/cplusplus/lumenflinger.md]: By the way, use "客户端" to replace the "client", use the Chinese if possible, thank you :)

File: development/cplusplus/lumenflinger.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ m_fps：服务端的刷新频率，目前最大30fps,过快刷新灯光数据可
 
 ## lumeneffects
 
-lumeneffects基于lumenflinger服务提供了一些简单的灯光效果接口。代码位于主要接口如下：（由LumenEffects.cpp、LumenEffects.h定义）。
+lumeneffects基于lumenflinger服务提供了一些简单的灯光效果接口。代码的主要接口如下：（由LumenEffects.cpp、LumenEffects.h定义）。
```
Comment (yorkie) [inline development/cplusplus/lumenflinger.md]: Change to "接口如下"。

File: development/cplusplus/lumenflinger.md [hunk 3]
```diff
@@ -183,7 +183,7 @@ lumeneffects基于lumenflinger服务提供了一些简单的灯光效果接口
 
 - **void InitLayers\(int speed\)**
 
-  初始化图层信息，在目前版本中负责设定动画的延迟，可用于自定义动画。可在以后增添内容
+  初始化图层信息，在目前版本中负责设定动画的延迟，用于自定义动画。以后会增添内容
```
Comment (yorkie) [inline development/cplusplus/lumenflinger.md]: Change to "初始化图层信息，在目前版本中负责设定动画的延迟、自定义 LED 动画"。

File: development/cplusplus/mediaplayer.md [hunk 4]
```diff
@@ -10,7 +10,7 @@
 
 ## **librplayer**
 
-目前请各开发者、厂商使用 C++ 接口，C 的接口定义的不是很友好，我们会对接口重新设计，但对 C++ 接口是透明的。
+目前请各开发者、厂商使用 C++ 接口，C 的接口定义不是很友好，我们会对接口重新设计，但对 C++ 接口是透明的。
```
Comment (yorkie) [inline development/cplusplus/mediaplayer.md]: Mark this line as a reference by using ">" in front of this line :)

File: development/cplusplus/wpa_ctrl.md [hunk 5]
```diff
@@ -51,7 +51,7 @@ int wifi_disable_all_network();
 
 ```
 
-我们同时提供了一个通用接口，我们上述所有接口均是通过这个接口来实现的。后续我们也会丰富更多的 WiFi 接口。
+同时我们提供了一个通用接口，上述所有接口均是通过这个接口来实现的。后续也会丰富更多的 WiFi 接口。
```
Comment (yorkie) [inline development/cplusplus/wpa_ctrl.md]: Remove the "后续也会丰富更多的 WiFi 接口", and don't tell developers what's our plan is provide more APIs.

File: development/cplusplus/wpa_ctrl.md [hunk 6]
```diff
@@ -77,7 +77,7 @@ int wifi_send_command(int cmd, void *value, int val_len, void *res, int res_len)
 
 ## AP模式API
 
-对于目前支持 ROKIDOS 的几款官方开发板，采用的 WiFi 模组是博通公司的 AP6255 芯片，博通公司的 wifi 芯片 AP 与 STATION 切换需要对网卡驱动进行卸载重装，所以配网方式不建议使用 AP 模式配网，这会造成多次 WiFi 模式的切换，耗时可能比较严重。不过给出以下方法，开发者可以自行配置，进入 AP 模式。
+对于目前支持 RokidOS 的几款官方开发板，采用的 WiFi 模组是博通公司的 AP6255 芯片，博通公司的 wifi 芯片 AP 与 STATION 切换需要对网卡驱动进行卸载重装，所以配网方式不建议使用 AP 模式配网，这会造成多次 WiFi 模式的切换，耗时可能比较严重。不过给出以下方法，开发者可以自行配置，进入 AP 模式。
```
Comment (yorkie) [inline development/cplusplus/wpa_ctrl.md]: I will suggest to use link on these words: "博通", "AP6255", and etc., and mark this as a ref either, thank you :)

File: development/cplusplus/wpa_ctrl.md [hunk 7]
```diff
@@ -134,9 +134,9 @@ dnsmasq -iwlan0  --dhcp-option=3,192.168.2.1 --dhcp-range=192.168.2.50,192.168.2
 
 RokidOS 配套的开发板支持 WiFi 连接 internet ，系统提供了一些控制 WiFi 连接的接口，配网业务及网络监控服务。设备第一次连接网络需要第三方设备发送 WiFi 相关信息如 SSID ， PSK 和加密方式到设备，第一次联网方式支持如下方式：
 
-### 蓝牙配网
+### 1. 蓝牙配网
```
Comment (yorkie) [inline development/cplusplus/wpa_ctrl.md]: Remove  this change.

File: development/cplusplus/wpa_ctrl.md [hunk 8]
```diff
@@ -134,9 +134,9 @@ dnsmasq -iwlan0  --dhcp-option=3,192.168.2.1 --dhcp-range=192.168.2.50,192.168.2
...
 
-### AP模式配网
+### 2. AP模式配网
```
Comment (yorkie) [inline development/cplusplus/wpa_ctrl.md]: If you wanna group the bluetooth/AP configuring networks, you should do like this:

```md
### 配网
balabala....
#### BLE模式
balabala....
#### AP模式
balabala....
```

File: porting/bluetooth/bluetooth.md [hunk 9]
```diff
@@ -29,7 +29,7 @@ robot/services/btflinger：以上代码包实际所包含的代码所在
 * 删掉 output/banban\_m\_a113/build/btflinger/
 * 再重新make
 
-# 如果模组相同，只是迁移平台：
+## 如果模组相同，只是迁移平台：
```
Comment (yorkie) [inline porting/bluetooth/bluetooth.md]: This is a title marked by the "##".

**Pull request merged**